src/routes/safe/components/Apps/index.jsx

```diff
@@ -0,0 +1,140 @@
+// @flow
+import React, { useState, useEffect, useCallback } from 'react'
+import styled from 'styled-components'
+
+import sendTransactions from './sendTransactions'
+
+const AppsWrapper = styled.div`
+  margin: 15px;
+`
+
+const operations = {
+  SEND_TRANSACTIONS: 'sendTransactions',
+  GET_TRANSACTIONS: 'getTransactions',
+  ON_SAFE_INFO: 'onSafeInfo',
+  ON_TX_UPDATE: 'onTransactionUpdate',
+}
+
+type Props = {
+  web3: any,
+  safeAddress: String,
+  network: String,
+  createTransaction: any
+}
+
+function Apps({
+  web3, safeAddress, network, createTransaction,
+}: Props) {
+  const iframeUrl = 'http://localhost:3002'
```

Comment:
Hardcoded for now, until we define where/how the third-party apps will be persisted. (URL, name, etc)

Reply:
If you want to test it locally, you can add any URL here. I will create the third-party project for Compound and ENS soon. Tomorrow perhaps.